requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible-core>=2.12 # require undef filter
+ansible-core>=2.12,<2.19 # require undef filter, pin below 2.19 until templating fixed
```

Comment:
Nit, do we have the error (or issue on GH) to track when we can unpin?
It doesn't need changing on this PR, but it would be good to throw it in somewhere so we know where to look if we're at the point of unpinning